examples/camera/camera_orbit.rs

```diff
@@ -1,22 +1,49 @@
 //! Shows how to orbit camera around a static scene using pitch, yaw, and roll.
+//!
+//! See also: `first_person_view_model` example, which does something similar but as a first-person
+//! camera view.
 
 use std::{f32::consts::FRAC_PI_2, ops::Range};
 
-use bevy::prelude::*;
+use bevy::{
+    input::mouse::{AccumulatedMouseMotion, MouseButtonInput},
+    prelude::*,
+};
 
-#[derive(Debug, Default, Resource)]
+#[derive(Debug, Resource)]
 struct CameraSettings {
     pub orbit_distance: f32,
-    // Multiply keyboard inputs by this factor
-    pub orbit_speed: f32,
+    pub pitch_speed: f32,
     // Clamp pitch to this range
     pub pitch_range: Range<f32>,
+    pub roll_speed: f32,
+    pub yaw_speed: f32,
+}
+
+impl Default for CameraSettings {
+    fn default() -> Self {
+        let pitch_limit = FRAC_PI_2 - 0.01;
+        Self {
```

Comment:
Mention that these are arbitrary sensitivities so that users aren't wondering where these numbers come from